src/sleep.ts

```diff
@@ -1,3 +1,3 @@
-export default function sleep(ms: number) {
```

Comment:
Do we need this sleep function outside of the tests as well? 🤔

Reply:
Yep, to wait for the correct block time (ie, refunds)